deploy/service_account.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: devopsconsole-operator
+  name: devopsconsole-operator
```

Comment:
Please change your editor configuration not to remove the newline.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795

Reply:
Indeed, let me fix that.
@baijum I found that cross-IDE formatting: https://editorconfig.org/ that may help. wdyt?